continuous_integration/scripts/test_report.py

```diff
@@ -48,17 +48,17 @@ def parse_args(argv: list[str] | None) -> argparse.Namespace:
         help="github events",
     )
     parser.add_argument(
-        "--days",
+        "--max-days",
```

Comment:
Looks like you've changed the arg name here, but not in the actual invocation in the workflow.